README.adoc

```diff
@@ -107,7 +107,7 @@ include::/complete/src/main/java/hello/BatchConfiguration.java[tag=readerwriterp
 The first chunk of code defines the input, processor, and output.
 - `reader()` creates an `ItemReader`. It looks for a file called `sample-data.csv` and parses each line item with enough information to turn it into a `Person`.
 - `processor()` creates an instance of our `PersonItemProcessor` you defined earlier, meant to uppercase the data.
-- `write(DataSource)` creates an `ItemWriter`. This one is aimed at a JDBC destination and automatically gets a copy of the dataSource created by `@EnableBatchProcessing`. It includes the SQL statement needed to insert a single `Person` driven by Java bean properties.
+- `writer()` creates an `ItemWriter`. This one is aimed at a JDBC destination and automatically gets a copy of the dataSource created by `@EnableBatchProcessing`. It includes the SQL statement needed to insert a single `Person` driven by Java bean properties.
```

Comment:
Per [here](https://github.com/spring-guides/gs-batch-processing/blob/master/complete/src/main/java/hello/BatchConfiguration.java#L56), it really should be `writer(DataSource)`, right?

Reply:
The instance variable "dataSource" was removed in https://github.com/spring-guides/gs-batch-processing/commit/1991d56d1b89fc10752b8242463a6d1a3fafa722#diff-d7bb631b000e17fee456d308d018f438L35 so this PR is not correct anymore. I'm closing it.